language/e2e-tests/src/tests/validator_set_management.rs

```diff
@@ -63,6 +63,24 @@ fn validator_add() {
         .events()
         .iter()
         .any(|e| e.key() == &new_epoch_event_key()));
+
```

Comment:
I think this would be easier to read/maintain as a functional test instead of an E2E test (now that we have functional test support for scripts). Check out https://github.com/libra/libra/tree/master/language/move-lang/functional-tests/tests/stdlib_scripts for examples

Reply:
Done

Reply:
I think Sam was proposing that we not keep this. Personally I don't really mind either way. I think we should be comfortable deleting "duplicate" tests if we're doing a refactor.